app/views/registrations/_accept_privacy_policy.html.erb

```diff
@@ -4,10 +4,11 @@
   <p class="govuk-body">Our privacy notice explains how we use your personal data. It is important that you have read this notice before continuing.</p>
 
   <div class="govuk-form-group">
-    <a href='https://www.gov.uk/help/privacy-notice'>Privacy policy</a>
+  <!-- only permit latest privacy policy acceptance -->
+    <%= link_to "Privacy policy", privacy_policy_path %>
   </div>
 
-  <%= f.govuk_check_box :accepted_policy_id, true, multiple: false, label: { text: "Accept the privacy policy"} %>
+  <%= f.govuk_check_box :accepted_policy_id, ApiClient.get_latest_privacy_policy.id, multiple: false, label: { text: "Accept the privacy policy"} %>
```

Comment:
This should be tied to the policy they are actually accepting - currently if the policy changes whilst they are viewing  the accept policy step they will be marked as accepting a policy we did not link them to. This is how we do it in the app - store the policy id in a hidden field and use the same one to render the link to the policy (the route to view the privacy policy then needs to accept an id to show a specific version, or default to the latest):

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/event_steps/_further_details.html.erb#L4

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/event_steps/_further_details.html.erb#L29

Reply:
I wrote it that way originally, then when I spoke with Myles this morning he said we always want the candidate to agree to just the latest policy. I can revert it after lunch.

Reply:
I have now put the id into the session, been back and forth with the team over this, I think this is fit for purpose.